packages/vitest/README.md

```diff
@@ -1,4 +1,23 @@
 # @effection/vitest
+
+## config
+
+`@effection/vitest` needs to be inlined to avoid node resolution problems and vitest version discrepencies:
```

Comment:
Can we say more about this and/or link to vite/vitest docs about it? It seemed to be working before, but is not working now? I think it would be helpful to have the "why" behind this bit of config.

Reply:
can you show me it working in an external repo from something which is not this branch?

Reply:
This was the original spike that I did outside of the repo.
https://github.com/jbolda/covector/tree/vitest-spike